contrib/packaging/docker/Dockerfile.runtime

```diff
@@ -58,7 +58,7 @@ cd llvm/ && \
 git checkout -b d941df363d1cb621a3836b909c37d79f2a3e27e2 d941df363d1cb621a3836b909c37d79f2a3e27e2 && \
 cd llvm/build && \
 cmake .. -G "Ninja" -DLLVM_TARGETS_TO_BUILD="BPF" -DLLVM_ENABLE_PROJECTS="clang" -DBUILD_SHARED_LIBS=OFF -DCMAKE_BUILD_TYPE=Release -DLLVM_BUILD_RUNTIME=OFF && \
-ninja && \
+ninja clang llc && \
```

Comment:
Needs a new image tag for the runtime image. Do you want to do it part of this PR?

Reply:
Thanks, forgot about this. Will trigger a new build and tag the image.